libafl_qemu/Cargo.toml

```diff
@@ -67,7 +67,7 @@ addr2line = "0.21"
 typed-arena = "2.0"
 paste = "1"
 enum-map = "2.7"
-
+serde_yaml = "0.8"
```

Comment:
Maybe we should hide the injection feature behind a `cfg` feature flag and make the yaml feature optional? No reason to pull in yaml for 99% of runs.
(Also, I'm still team TOML :) )

Reply:
Then nobody will know about the feature and it wont be used. then we can basically close this PR.

Reply:
Why? We can still make the example fuzzer use it, but we don't bloat the every-day usecase with yaml libs